File: Begginers/Python/quadrante_1115.py
while True:
      XY = input().split()
      X = int(XY[0])
      Y = int(XY[1])
      if X > 0 and Y > 0:
         print(f"primeiro")
      elif X < 0 and Y > 0:
         print(f"segundo")
      elif X < 0 and Y < 0:
         print(f"terceiro")
      elif X > 0 and Y < 0:
         print(f"quarto")
      else:
           break

File: Begginers/Java/quadrante_1115.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
       public static void main(String[] args) throws IOException {
              Scanner sc = new Scanner(System.in);

              int x, y;

              while (true) {
                     x = sc.nextInt();
                     y = sc.nextInt();

                     if (x == 0 || y == 0) {
                            break;
                     } else if (x > 0 && y > 0) {
                            System.out.println("primeiro");
                     } else if (x > 0 && y < 0) {
                            System.out.println("quarto");
                     } else if (x < 0 && y > 0) {
                            System.out.println("segundo");
                     } else if (x < 0 && y < 0) {
                            System.out.println("terceiro");
                     }
                     
              }
      
              sc.close(); 

       }
}
